src/Etg.Yams.Core/Process/Process.cs

```diff
@@ -65,22 +65,26 @@ public Process(string exePath, string exeArgs, bool showProcessWindow)
                     _process.StartInfo.UseShellExecute = false;
                     _process.StartInfo.CreateNoWindow = true;
                 }
+
                 if (!_process.Start())
                 {
                     await ReleaseResources();
-                    throw new Exception($"The OS failed to start the process at {_exePath} with arguments {_exeArgs}");
+                    throw new Exception(
+                        $"The OS failed to start the process at {_exePath} with arguments {ExeArgs}");
                 }
+
                 _isRunning = true;
             });
         }
 
-        public async Task Close()
+    public async Task Close()
         {
             if (_process == null) return;
 
             _process.Exited -= ProcessExited;
             await Task.Run(() =>
             {
+                //TODO: Use Process.WaitForExit to wait for the process to exit
```

Comment:
Yes. This isn't a bug, we are currently spin waiting for the process to exit (by checking the Exited flag) but it would be simpler to just use the `WaitForExit` API.